var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -164,6 +164,8 @@ class Mfem(Package):
 
     depends_on('mpi', when='+mpi')
     depends_on('hypre', when='+mpi')
```

Comment:
you will also want to restrict to a newer hypre for newer versions, or is this incompatibility only one-way?

Reply:
The current `@develop` version works fine with all hypre versions - no need to restrict to 2.14 and above.